tests/storage/utils.py

```diff
@@ -116,6 +116,14 @@ def upload_token_request(storage_ns_name, pvc_name, data, client):
                 break
 
 
+@contextmanager
+def ensure_vm_running(vm, stop_vm=True):
+    running_vm(vm=vm)
+    yield vm
+    if stop_vm:
+        vm.stop(wait=True)
```

Comment:
This will stop the VM only on teardown? I think we can remove this function and stop the VM in the test flow where needed, making it easier to follow the test flow.